src/uWS.cpp

```diff
@@ -1,3 +1,7 @@
+#if HAVE_WIN32_CPP_CLI
```

Comment:
@alexhultman Added a header guard preventing people from fucking up by embedding uWS directly into their project. Though i seriously doubt there will be anyone but me :laughing: